### DISCUSSION TITLE
Getting Jaeger client to work in Java 6 environment

### DISCUSSION THREAD

**Discussion by lam1234567** [jaegertracing/jaeger#3338] [Q&A]
Hi everyone,

We are trying to set up Jaeger Client on instrument an application in a Java 6 environment to send spans to Jaeger collector's http endpoint.  We found that the Jaeger Client depends on okhttp3 and okhttp3 (even its oldest version 3.0.0.) doesn't support Java 6.  Can someone shed some light on how we can get this to work?

Thanks a lot.

**Comment by yurishkuro** [comment]
You can use one of the older versions of Jaeger client. We no longer support Java 6.

**Reply by lam1234567** [reply]
Thanks for your reply Yuri.  I tried older versions of Jaeger client but I cannot find one version that supports both Java 6 And sending spans directly to Jaeger Collector via http.

It seems that support for sending spans directly to Jaeger endpoint and dependency on okhttp3 (which doesn't work with Java 6) are added in the same PR: https://github.com/jaegertracing/jaeger-client-java/commit/9c6d3a5d1f8b1cd1482bce37d7695ac6903e71e1
